models/ac_stark.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+from itertools import product
+from arc import DynamicPolarizability, ShirleyMethod
+from arc import Cesium as Cs
+from models.utility import wavelength2freq, power2field, p2i_gaussian, w2omega, E2f, E2T, Rayleigh_range, trap_frequency
```

Comment:
When importing this many functions, you may as well import models.utility as util and then just call util.function()